code/modules/vehicles/wheelchair.dm

```diff
@@ -108,6 +119,8 @@
 	usr.visible_message(span_notice("[usr] collapses [src]."), span_notice("You collapse [src]."))
 	var/obj/vehicle/ridden/wheelchair/wheelchair_folded = new foldabletype(get_turf(src))
 	usr.put_in_hands(wheelchair_folded)
+	if(bell_attached)
+		remove_bell()
```

Comment:
This shouldn't be necessary, since we're deleting the wheelchair here, it should call Destroy() which will drop it already.